dali/kernels/slice/slice_gpu.cuh

```diff
@@ -237,14 +240,22 @@ class SliceGPU {
 
     std::vector<int64_t> sample_sizes;
     sample_sizes.reserve(slice_args.size());
+    int64_t total_volume = 0;
     for (auto &args : slice_args) {
       sample_sizes.push_back(volume(args.shape));
+      total_volume += volume(args.shape);
+    }
+
+    auto minBlocks = 4 * GetSMCount();
```

Comment:
We don't use camelCase. Also, don't use auto for trivial types.
```suggestion
    int min_blocks = 4 * GetSMCount();
```

Reply:
Fixed